packages/squiggle-lang/__tests__/TS/JS_test.ts

```diff
@@ -56,16 +56,16 @@ describe("Distribution", () => {
   );
 
   test("mean", () => {
-    expect(dist.mean().value).toBeCloseTo(3.737);
+    expect(dist.mean().value).toBeCloseTo(5.3913);
```

Comment:
Again, good change. Would have been even better to have caught this a bit earlier

Reply:
Yea, that was kind of scary.